asdf_compression/blsc/compressor.py

```diff
@@ -7,10 +7,34 @@ class BloscCompressor(Compressor):
     def compress(self, data, **kwargs):
         import blosc
 
-        yield blosc.compress(data, **kwargs)
+        # Type size, necessary for shuffle filters
+        typesize = data.itemsize
+
+        # Compression level
+        clevel = kwargs.get("level", 9)
+
+        # Shuffle filter
+        shuffle_string = kwargs.get("filter", "bitshuffle")
```

Comment:
Similar to `level` vs `clevel`, I'd say we avoid adding a layer that remaps arguments. Without the remapping we can document this as "kwargs is anything supported by blosc.compress" (or something similar). If we map arguments we'll want to document the new mapping.